proposals/esm-integration/README.md

```diff
@@ -29,121 +29,112 @@ WebAssembly
 A declarative API would improve the ergonomics by making this work happen implicitly.
 
 ```js
-import {foo} from "./myModule.wasm";
+import { foo } from "./myModule.wasm";
 foo();
 ```
 
-### Enable WebAssembly modules to take part in JavaScript module graphs
-
-With the introduction of ES modules, the JS spec provides a way to express module dependency graphs.
-
-These graphs are expressed with import and export statements.
+Then by integrating with [Source Phase Imports](https://github.com/tc39/proposal-source-phase-imports), arbitrary instantiations with custom imports can still be supported.
 
 ```js
-// main.js
-
-import {count, incrementCount} from "./dep.js"
+import source myModule from "./myModule.wasm");
+const { foo: foo1 } = await WebAssembly.instantiate(myModule, { ...imports });
+const { foo: foo2 } = await WebAssembly.instantiate(myModule, { ...imports });
```

Comment:
Was the intention here to use a different set of imports?

Reply:
This was mainly trying to demonstrate multi-instantiation, but I've distinguished the imports as well.